packages/block-library/src/gallery/editor.scss

```diff
@@ -198,3 +198,11 @@
 	justify-content: flex-end;
 	gap: $grid-unit-15;
 }
+
+.blocks-gallery__link-to-control {
+	.components-menu-item__item {
+		.components-menu-item__info {
+			white-space: nowrap;
+		}
+	}
+}
```

Comment:
Yes, the info text for the lightbox option used to be two lines long. However, we adjusted the text to make it shorter and it now stays in a single line without this CSS. I will remove the CSS because it is no longer useful.